packages/server/lib/browsers/chrome.ts

```diff
@@ -375,6 +375,7 @@ export = {
 
     // copy the extension src to the extension dist
     await utils.copyExtension(pathToExtension, extensionDest)
+    fs.chmodSync(extensionBg, 0o0644)
```

Comment:
Use asynchronous methods:

```suggestion
    await fs.chmod(extensionBg, 0o0644)
```